lib/Transforms/IPO/FunctionImport.cpp

```diff
@@ -102,6 +102,12 @@ static cl::opt<bool> PrintImports("print-imports", cl::init(false), cl::Hidden,
 static cl::opt<bool> ComputeDead("compute-dead", cl::init(true), cl::Hidden,
                                  cl::desc("Compute dead symbols"));
 
+bool llvm::ForceWeakImportFlag = false;
+static cl::opt<bool, true>
+ForceWeakImport("force-weak-import", cl::Hidden,
```

Comment:
just a nit, could we rename the flags to `force-import-weak` instead?  We'd want to import weak functions and not performing a weaker import... :)

Reply:
Good catch!